pkg/cmd/roachtest/tests/large_schema_backup_restore.go

```diff
@@ -56,6 +56,9 @@ func largeSchemaBackupRestore(r registry.Registry, numTables int) {
 			conn := c.Conn(ctx, t.L(), 1)
 			defer conn.Close()
 
+			_, err := conn.ExecContext(ctx, "SET autocommit_before_ddl = false")
```

Comment:
might be worth a comment mentioning that creating the objects in one transaction is significantly faster

Reply:
done!

Reply:
`conn` here is a `*sql.DB` right, which is backed by a conn _pool_ isn't it? Do we need to `SET` in the tx to be sure it applies to the right conn?

Reply:
oof, is it really? lemme open a new pr